external/js/cothority/src/byzcoin/proof.ts

```diff
@@ -372,6 +329,19 @@ class InclusionProof extends Message<InclusionProof> {
     get value(): Buffer {
         return this.leaf.value;
     }
+
+    /**
+     * Returns the stateChangeBody of this InclusionProof.
+     */
+    get stateChangeBody(): StateChangeBody {
+        if (!this._state) {
```

Comment:
```suggestion
        if (this._state === undefined) {
```

Reply:
nah, as now it's either undefined or an instance of `StateChangeBody`